doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -110,41 +113,53 @@ The presence of a ``bicr`` node in the application devicetree will automatically
 Flashing the BICR binary
 ========================
 
+##TODO
+
 After the |NCS| build system generates the BICR binary, you must flash this binary manually.
 The content of BICR should be loaded to the SoC only once and should not be erased nor modified unless the PCB layout changes.
-To manually flash the generated :file:`bicr.hex` file to the SoC, use nRF Util as follows::
+To manually program the generated :file:`bicr.hex` file to the SoC, use nRF Util as follows::
+
+    nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware bicr.hex --core Application --serial-number <serial_number>
+
+You only need to follow this programming process once, assuming the PCB configuration applied through the BICR is correct the first time.
+However, it is also possible to reprogram the BICR while in the LCS ``RoT``.
+This can be useful, for example, when adjusting the configuration as the PCB design gets refined or modified, requiring the process to be repeated.
+
+Validate the BICR binary
+------------------------
+
+After programming the BICR binary onto the device, you should validate whether the BICR was successfully installed and executed.
 
-    nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware bicr.hex` --core Secure --serial-number <serial_number>
+Reset the device
+   The simplest way to check if the BICR is functioning is to reset the device.
+   If the device is still operational after the reset, the BICR is at least partially loaded.
 
-You need to follow this flashing process only one time, as the PCB configuration will not change.
+Validate using nrfutil
+  When in LCS ``EMPTY``, you can use `nrfutil` to explicitly verify the BICR status by reading the memory::
+
+     nrfutil device x-read --address 0x2F88FF1C --serial-number 1051164514 --core Secure
+
+  nrfutil returns the BICR loading status using one of the following values:
+
+  * ``0x289CFB73``: BICR loaded without error.
+  * ``0xD78213DF``: No BICR was loaded (as none was found during the boot).
+  * ``0xCE68C97C``: BICR loading failed (device can be unresponsive and not being able to print this value).
```

Comment:
If a user gets any of these error codes, we should add some instructions on what to do then.